### PR TITLE
Fix S3 tests

### PR DESCRIPTION
**Pull request by kokosing**

### CODE REVIEW

**Comment by kokosing** [comment]
We had a bug in our infra, something similar to https://github.com/prestodb/presto/pull/10662. That bug caused that build status was the same as last test script command status.

We fixed that and we see that automation for master fails. Here is the fix, I think that there might be more bugs in tests because of that bug.

CC: @electrum

**Comment by kokosing** [comment]
I am still testing... so far I caught 3 issues.

**Comment by kokosing** [comment]
That it is all, now tests are passing on our internal infra.

Notice that second commit comes from: https://github.com/prestodb/presto/pull/10669

**Comment by findepi** [comment]
> Notice that second commit comes from: #10669

rebase? #10669 is merged.

**Comment by findepi** [comment]
oh, not merged, just closed.

File: presto-hive-hadoop2/bin/run_hive_s3_tests.sh [hunk 1]
```diff
@@ -23,6 +23,10 @@ exec_in_hadoop_master_container /usr/bin/hive -e "CREATE EXTERNAL TABLE presto_t
 
 stop_unnecessary_hadoop_services
 
+# restart hive-metastore to apply S3 changes in core-site.xml
```
Comment (findepi) [inline presto-hive-hadoop2/bin/run_hive_s3_tests.sh]: Would it be possible to apply the s3-related changes before starting the container?

File: .travis.yml [hunk 2]
```diff
@@ -155,7 +155,7 @@ script:
     if [[ -v HIVE_TESTS && -v HIVE_TESTS_AWS_ACCESS_KEY_ID ]]; then
       env AWS_ACCESS_KEY_ID=$HIVE_TESTS_AWS_ACCESS_KEY_ID \
         AWS_SECRET_ACCESS_KEY=$HIVE_TESTS_AWS_SECRET_ACCESS_KEY \
-        ./mvnw -pl presto-hive test -P test-hive-glue
+        ./mvnw -pl presto-hive test -B -P test-hive-glue
```
Comment (findepi) [inline .travis.yml]: Sounds good, but I am curious -- what problem does it fix?

Comment (kokosing) [reply .travis.yml]: It actually does not fix anything, it just makes the output less verbose (no progress is logged for artifacts downloading)

File: presto-hive-hadoop2/bin/run_hive_s3_tests.sh (hunk 1 above)
Comment (kokosing) [reply presto-hive-hadoop2/bin/run_hive_s3_tests.sh]: Then the change would be much more invasive.

File: .travis.yml (hunk 2 above)
Comment (findepi) [reply .travis.yml]: Add this to the commit msg

**Pull request merged**